docs/submitters/submissionGuide.md

```diff
@@ -8,6 +8,9 @@ Submitting an add-on version is done via a GitHub issue form.
 A JSON metadata file is generated from the issue form and the add-on's manifest.
 A detailed description of the JSON schema can be found in [jsonMetadata.md](./jsonMetadata.md).
 When the issue form is submitted, the JSON file is generated and submitted as a pull request to the repository.
+Security analysis is performedfor the submitted add-on.
+If there are detected vulnerabilities, a link to a GitHub workflow will be shown in the issue. You can open this link and download artifacts containing the results of the analysis.
+Unzip artifacts and open the .sarif file in your preferred editor. Then, you can search properties like "region" to investigate where vulnerabilities are located in your code.
```

Comment:
I really think reading the RAW sarif format is challenging for users.
I have suggested an alternative, a web based sarif viewer from microsoft. It appears accessible with NVDA+Chrome.

```suggestion
If there are detected vulnerabilities, a link to a GitHub workflow will be shown in the issue.
You can open this link and download artifacts containing the results of the analysis.
Unzip artifacts and open the .sarif file in your preferred editor.
For example, you can use [Microsoft's Sarif web based reader](https://microsoft.github.io/sarif-web-component/).
```